src/pool/srv_target.c

```diff
@@ -1031,6 +1031,92 @@ ds_pool_tgt_query_handler(crt_rpc_t *rpc)
 	crt_reply_send(rpc);
 }
 
+static int
+fetch_and_connect(struct ds_pool *pool, uuid_t handle_uuid)
+{
+	struct pool_iv_conn *conn_args;
+	d_iov_t conn_args_iov;
+	int rc;
+
+	/* This buffer will likely be reallocated by the IV fetch operation
+	 * in the event that the cred data is nonzero size
+	 * Thus because it might be realloc'd, it needs to not be stack
+	 * allocated
+	 */
+	D_ALLOC_PTR(conn_args);
+	if (conn_args == NULL)
+		D_GOTO(out, rc=-DER_NOMEM);
```

Comment:
(style)  spaces required around that '=' (ctx:VxO)
(style)  space required before that '-' (ctx:OxV)